src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LightCompiler.cs

```diff
@@ -821,12 +821,12 @@ private void CompileBinaryExpression(Expression expr)
                             if (node.NodeType == ExpressionType.Equal)
                             {
                                 // right null, left not, false
-                                _instructions.EmitLoad(ScriptingRuntimeHelpers.Boolean_False, typeof(bool));
+                                _instructions.EmitLoad(AstUtils.BoxedFalse, typeof(bool));
```

Comment:
`Utils` was already imported into this file as `AstUtils` so I stayed consistent with the file. I don't see much point in the alias as there's no other type of that name IIRC. I wouldn't mind going either way if we wanted to be consistently `Utils` or consistently `AstUtils`.